makefile

```diff
@@ -1,7 +1,7 @@
-# replace this variable with the location that leads to your air executable file
-air_location=~/go/bin/air 
+GO_BIN=~/go/bin
+
 dev:
-	${air_location}
+	${GO_BIN}/air
```

Comment:
The issue with this is that we need to make sure everyone has their air file in their go/bin directory. I am totally down to mandate having it in go/bin directory though

Reply:
We'll, the way it was before with ${air_location} has the same issue. So this isn't really apart of this issue. We can discuss this at our meeting, but i'm resolving this conversation in this pr.